po/cs.po

```diff
@@ -116,21 +116,23 @@ msgid "Grant Access"
 msgstr "Schválit přístup"
 
 #: src/location.c:535
-msgid "Grant Access to Your Location?"
+#, fuzzy
+msgid "Allow Access to Your Location?"
 msgstr "Schválit přístup k vaší poloze?"
 
 #: src/location.c:536
-msgid "An application wants to use your location."
+#, fuzzy
+msgid "An application wants to known the location of this device."
```

Comment:
The l10n stuff presumably needs to be regenerated for the updated string.

Reply:
"wants to known" should be "wants to know"

But I disagree that this is clearer. Also, it is longer.